### ISSUE TITLE
Show Compo/Jam in rating area

### ISSUE THREAD

**Issue by Cerno-b** [ludumdare/ludumdare#966]
The old site made made a difference in the rating area between **Overall** and **Overall (Jam)** to clearly indicate what category the game took place in. Several people have apparently confused my game for a Jam game. While I was honored by that misconception, I feel that my ratings would be worse than when people were knowingly rating a Compo game.

So since the only way to know whether it's Compo or Jam is the game's header and it's not really in your face when voting, I would like to have the old way of marking Compo and Jam back. Maybe we can have **Overall (Jam)** and **Overall (Compo)** to be fair, but marking it in the exact place where you vote is unmistakable.

**Comment by local-minimum** [comment]
![gametype](https://cloud.githubusercontent.com/assets/8041100/26275182/6128e21c-3d5b-11e7-8644-9d2ef4b36dfa.png)

You do have it at the top, but I agree that the information is a bit too far away from the voting bit. Personally I don't think too much about which category the game is in, in part it doesn't matte too much either because the ratings are done separately.

**Comment by Cerno-b** [comment]
Yeah, I realize that it is up there (that's what I meant by "game's header"), but as you say, there is usually a lot of text between header and voting.

I just now realized that the voting does have the word "Compo" or "Jam". However I find it easy to skim over, as the eyes naturally wander to the stars. If putting it behind the voting text like on the old page is too obtrusive, maybe you could make it one font size larger and/or bold face?

About it not mattering: I disagree. Everyone has their own metrics and if people hold a Compo game to the same standards as a Jam game, they would likely give the Compo game lower scores. If that wasn't the case, we wouldn't have to have separate evaluations. 

Look at it like this: Someone just voted for a Jam game with great extensive graphics, different awesome music tracks and a lot of polishing due to an extra day of work. Next they play a Compo game that has nice graphics, just one song and less content. I think it's easy to downvote such a game if people are not aware that it's a Compo entry. I often read comments like "impressive for a Compo game" in well-made Compo games, so that would mean that the same content for a Jam game would be "less impressive".

**Comment by povrazor** [comment]
It does say jam/compo in the subtext while ~~eating~~ rating.

**Comment by Cerno-b** [comment]
That's what I realized, but could it be made a bit more prominent? Like bold or one font size larger? The fact I hadn't noticed it was there at all kinda tells me that it's too small (or I am not attentive enough)

**Comment by zwrawr** [comment]
We could go with what i suggested in the original issue. It was crunch time so i think the easiest solution was taken. #840 
![image](https://cloud.githubusercontent.com/assets/18352787/26378035/adff8116-400a-11e7-8fb6-ef42a008ab64.png)

**Comment by Cerno-b** [comment]
Eff yeah! Since Compo and Jam are also color-coded that would be perfect!

I like the idea above as well, but wouldn't that clash with the screenshots that would appear right above the ratings section? I guess many people use an in-game screen shot as their thumbnail.

I think I like the second one better, can we have this please? 😀

**Comment by pfefferminzz** [comment]
I like the last mockup, too. Many users posted a screenshot of their final ratings everywhere. With COMPO / JAM directly next to Ratings it would be included in those screenshots more likely. That would be a nice side effect.
